examples/middleware/e2e/base.spec.ts

```diff
@@ -3,28 +3,28 @@ import { test, expect } from "@playwright/test";
 test("redirect", async ({ page }) => {
```

Comment:
This looks like a fix for https://github.com/opennextjs/opennextjs-cloudflare/issues/255

Would you mind extracting the code in a distinct PR for easier tracking?

Reply:
yes of course: https://github.com/opennextjs/opennextjs-cloudflare/pull/267, sorry I didn't realise there was an issue for this 👍

Reply:
(once we merge #267 I can simply rebase this PR)